tests/src/model/pages/images-page.ts

```diff
@@ -66,40 +75,33 @@ export class ImagesPage extends MainPage {
     if (await this.pageIsEmpty()) {
       return undefined;
     }
-    const table = await this.getTable();
-    const rows = await table.getByRole('row').all();
-    let first: boolean = true;
-    for (const row of rows) {
-      if (first) {
-        // skip first row (header)
-        first = false;
-        continue;
-      }
-      // test on empty row - contains on 0th position &nbsp; character (ISO 8859-1 character set: 160)
-      const zeroCell = await row.getByRole('cell').nth(0).innerText();
-      if (zeroCell.indexOf(String.fromCharCode(160)) === 0) {
-        continue;
-      }
-      const thirdCell = await row.getByRole('cell').nth(3).innerText();
-      const index = thirdCell.indexOf(name);
-      if (index >= 0) {
-        return row;
+    try {
+      const table = await this.getTable();
+      const rows = await table.getByRole('row').all();
+      for (let i = rows.length - 1; i > 0; i--) {
+        const thirdCell = await rows[i].getByRole('cell').nth(3).getByText(name).count();
```

Comment:
Also, might be a good idea to label the row inside .svelte providing ARIA functionality and also simplifying the test logic.

Reply:
That's a fair point, will change it to getByText with exact param enabled.

Reply:
Actually checking exactly will no be possible for now on images without further refactoring, since when e're checking for an image on the page we're looking for a substring inside the tests, not the entire name, but I have changed it to exact for containers and pods.

Reply:
ok changed images, containers and pods to be matched exactly when looking in the table